src/app/zap-templates/zcl/data-model/chip/microwave-oven-control-cluster.xml

```diff
@@ -25,17 +31,23 @@ limitations under the License.
     <define>MICROWAVE_OVEN_CONTROL_CLUSTER</define>
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
-    <attribute side="server" code="0x0001" define="COOK_TIME" type="elapsed_s" default="30" writable="false" optional="false">CookTime</attribute>
-    <attribute side="server" code="0x0002" define="POWER_SETTING" type="int8u" default="100" writable="false" optional="false">PowerSetting</attribute>
-    <attribute side="server" code="0x0003" define="MIN_POWER" type="int8u" default="10" writable="false" optional="true">MinPower</attribute>
-    <attribute side="server" code="0x0004" define="MAX_POWER" type="int8u" default="100" writable="false" optional="true">MaxPower</attribute>
-    <attribute side="server" code="0x0005" define="POWERSTEP" type="int8u" default="10" writable="false" optional="true">PowerStep</attribute>
+    <attribute side="server" code="0x0000" define="COOK_TIME"           type="elapsed_s" default="30"  writable="false" optional="false">CookTime</attribute>
+    <attribute side="server" code="0x0001" define="MAX_COOK_TIME"       type="elapsed_s"               writable="false" optional="false" min="0x00" max="86400">MaxCookTime</attribute>
+    <attribute side="server" code="0x0002" define="POWER_SETTING"       type="int8u"     default="100" writable="false" optional="true">PowerSetting</attribute>
+    <attribute side="server" code="0x0003" define="MIN_POWER"           type="int8u"     default="10"  writable="false" optional="true">MinPower</attribute>
+    <attribute side="server" code="0x0004" define="MAX_POWER"           type="int8u"     default="100" writable="false" optional="true">MaxPower</attribute>
+    <attribute side="server" code="0x0005" define="POWER_STEP"          type="int8u"     default="10"  writable="false" optional="true">PowerStep</attribute>
+    <attribute side="server" code="0x0006" define="SUPPORTED_WATTS"     type="array" entryType="int16u" writable="false" isNullable="true" optional="true">SupportedWatts</attribute>
```

Comment:
This is not nullable in the spec.

Stopping here.  Please go through this XML and make sure it matches the spec, or use @hasty's tool for generating XML from the spec.

Reply:
@bzbarsky-apple May I as if there has the guidelines or link about the generating tool?

Reply:
@liangpy4 All I know is that it exists.  Please check with Hasty for details.